server/views/forms/forms-layout.pug

```diff
@@ -3,8 +3,7 @@ include includes/formsHeader
 doctype html
 html(lang="en")
   head
-    link(href= domain + "/public/stylesheets/forms-pdf.css?" + version, media="print", rel="stylesheet", type="text/css")
-
+    link(href= "http://127.0.0.1:3000/public/stylesheets/forms-pdf.css?" + version, media="print", rel="stylesheet", type="text/css")
```

Comment:
Is the localhost reference going to work in any other environment except locally?